src/link_prober/LinkProber.cpp

```diff
@@ -333,7 +333,7 @@ void LinkProber::sendHeartbeat(bool forceSend)
         mStream.write_some(boost::asio::buffer(mTxBuffer.data(), mTxPacketSize), errorCode);
 
         if (errorCode) {
-            MUXLOGTRACE(mMuxPortConfig.getPortName() + ": Failed to send heartbeat!");
+            MUXLOGTRACE(mMuxPortConfig.getPortName() + ": Failed to send heartbeat! Error code: " + errorCode.message());
```

Comment:
what's the output of `errorCode.message()`?

Reply:
A string, i.e. for `boost::system::errc::bad_file_descriptor` it will return "Bad file descriptor"